internal/pkg_npm/pkg_npm.bzl

```diff
@@ -149,16 +155,16 @@ def _filter_out_external_files(ctx, files, package_path):
                     result.append(file.path)
     return result
 
-def create_package(ctx, deps_sources, nested_packages):
+# Used in angular/angular /packages/bazel/src/ng_package/ng_package.bzl
+def create_package(ctx, files, nested_packages):
     """Creates an action that produces the npm package.
 
     It copies srcs and deps into the artifact and produces the .pack and .publish
     scripts.
 
     Args:
       ctx: the skylark rule context
-      deps_sources: Files which have been specified as dependencies. Usually ".js" or ".d.ts"
-                    generated files.
+      files: list of files to include in the package.
```

Comment:
will it be hard to roll out since it's a breaking change for ng_package rule?

Reply:
Nope. I had the same thought and I checked. Not using named params in angular repo.